### PR TITLE
Added instructions for deployment with Docker

### PR DESCRIPTION
**Pull request by nicole-brewer**
I'd still like to add a section about "Deploying with The Littlest JupyterHub and the Voila Dashboard Plugin"

### CODE REVIEW

**Comment by mwcraig** [comment]
Looks good so far!

File: 04a_deployment_serverside.ipynb [hunk 1]
```diff
@@ -96,7 +96,137 @@
...
+   "id": "9acd0c03-af60-459f-b61f-5f33db821d61",
+   "metadata": {},
+   "source": [
+    "### Takehome Exercise\n",
+    "\n",
+    "**Goal**: Build our Dockerfile and run the image on our local machine. Access our Voila app in our browser. \n",
+    "\n",
+    "1. [Install Docker](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository). Note that you may need admin access to run Docker commands.\n",
+    "2. Start a command line (terminal) session.\n",
+    "3. Make sure Docker by executing `docker info`.\n",
```
Comment (mwcraig) [inline 04a_deployment_serverside.ipynb]:
```suggestion
    "3. Make sure Docker is installed by executing `docker info`.\n",
```

**Review by mwcraig — APPROVED** [review]
Looks good - I haven't tried building the docker image yet

**Review by JuanCab — APPROVED** [review]
Looks good enough!

**Pull request merged**